student.rb

```diff
@@ -0,0 +1,29 @@
+require './main'
+
+# Public: Student class used for creating student objects
+#   Student class inherits from person class.
+#
+# All methods are class methods and should be called on the Person class.
+#
+# Examples
+#
+# student = Student.new('unknown')
+# student.play_hooky
+# # => '¯\(ツ)/¯'
+class Student < Person
+  attr_accessor :classroom
```

Comment:
- [ ] This `attr_accesor` is not needed for now since we are creating it in the constructor.